web/templates/task/index.html.eex

```diff
@@ -0,0 +1,33 @@
+<h2>Listing tasks</h2>
+
+<table class="table">
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Deadline</th>
+      <th>Status</th>
+      <th>Assigned to</th>
+    </tr>
+  </thead>
+  <tbody>
+<%= for task <- @tasks do %>
+    <tr>
+      <td><%= task.name %></td>
+      <td><%= task.deadline %></td>
+      <td><%= task.status %></td>
+      <%= if task.user != nil do %>
```

Comment:
I am not sure which one is the most used in Elixir, but `if task.user != nil do` is equivalent to `if task.user do`.

Coming from ruby the latter is more common